x-pack/platform/packages/shared/kbn-profiler-cli/src/get_node_processes.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import execa from 'execa';
+
+export async function getNodeProcesses(
+  grep?: string
+): Promise<Array<{ pid: number; command: string; ports: number[] }>> {
+  const candidates = await execa
+    .command(
+      `ps -eo pid,command | grep -E '^[[:space:]0-9]+[[:space:]]*node[[:space:]]?' | grep -v grep ${
```

Comment:
Does this command work cross platform?

Reply:
It definitely does not. I think that's ok because we suck at Windows support anyway.